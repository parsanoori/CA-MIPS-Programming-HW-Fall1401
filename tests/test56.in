63
115296577
802652231
40870478
682795375
401979016
24463664
834480694
923004832
168398135
991293721
539137152
654250689
64935285
306487580
1028972457
26894574
1046466031
357857850
914272473
944855515
732249616
968134184
509256903
300477288
693449280
960653891
169815885
294250825
426628766
882425739
464075360
463850419
754718041
33990239
184854404
85177332
708459583
946199771
543442883
575197969
259544041
1025944725
524319742
649530883
619174250
515144871
800867777
602631148
1049109853
829105879
1038725967
469564242
504249192
458606937
120219998
697374217
296805115
267861948
618178440
276177669
548316241
100211832
930895556