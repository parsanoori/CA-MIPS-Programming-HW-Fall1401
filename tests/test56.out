24463664
26894574
33990239
40870478
64935285
85177332
100211832
115296577
120219998
168398135
169815885
184854404
259544041
267861948
276177669
294250825
296805115
300477288
306487580
357857850
401979016
426628766
458606937
463850419
464075360
469564242
504249192
509256903
515144871
524319742
539137152
543442883
548316241
575197969
602631148
618178440
619174250
649530883
654250689
682795375
693449280
697374217
708459583
732249616
754718041
800867777
802652231
829105879
834480694
882425739
914272473
923004832
930895556
944855515
946199771
960653891
968134184
991293721
1025944725
1028972457
1038725967
1046466031
1049109853